docs/oldQuestion.md

```diff
@@ -26,7 +26,7 @@ See below for detailed information about each question type.
 
 - Example: [`fossilFuelsRadio`](https://github.com/PrairieLearn/PrairieLearn/tree/21be1db08978b6f8a8ca76c1e53681fe2ebe2fce/exampleCourse/questions/fossilFuelsRadio)
 - Required files: `info.json`
-- Options schema: [`MultipleChoice` options](https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/schemas/schemas/questionOptionsMultipleChoice.json)
+- Options Zod schema: [`MultipleChoice` options](https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/schemas/questionOptionsMultipleChoice.ts)
```

Comment:
I do think that, all things considered, the JSON schema is more suited for human consumption than the Zod schema.

Reply:
Agreed. In 66e3d0dcfa7c5ceddbd46b733714684bcad4c785 I point them at the user-facing schemas instead.